mkt/constants/applications.py

```diff
@@ -58,3 +58,5 @@ def get_device(request):
 
 def get_device_id(request):
     return getattr(get_device(request), 'id', None)
+
+HOMESCREEN = 'homescreen'
```

Comment:
Is that the tag name ? Weird place to put it, this file only contains devices info.